inclusive-design/wecount.inclusivedesign.ca
# feat: add support for image captions (resolves #1670)

## Description

Add support for image captions for regular image and image-and-text widgets.

## Steps to test

1. Go to the Admin page
2. Create a post with image or image-and-text widget
3. Provide title for the image
4. See the caption appears underneath the image



## Related issues

Resolves #1670
